src/shared/utils/upgradeForm.test.ts

```diff
@@ -130,9 +128,7 @@ describe('getDefaultValuesUpgradeForm', () => {
     })
 
     it('returns sentry year plan if user is sentry upgrade', () => {
-      const accountDetails = {
-        plan: { value: Plans.USERS_BASIC, quantity: 1 },
-      } as z.infer<typeof AccountDetailsSchema>
+      const accountDetails = {} as z.infer<typeof AccountDetailsSchema>
```

Comment:
could just define this once in the top of the file

Reply:
good point, cleaned it up in [91efa0a](https://github.com/codecov/gazebo/pull/3597/commits/91efa0ae3adf26d773cb6fbad71b111f3e7b02c1)